modules/caddytls/acmeissuer.go

```diff
@@ -349,6 +360,17 @@ func (iss *ACMEIssuer) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 
 	for d.NextBlock(0) {
 		switch d.Val() {
+		case "lifetime":
+			var lifetimeStr string
+			if !d.AllArgs(&lifetimeStr) {
+				return d.ArgErr()
+			}
+			lifetime, err := caddy.ParseDuration(lifetimeStr)
+			if err != nil {
+				return d.Errf("invalid lifetime %s: %v", lifetimeStr, err)
+			}
+			// TODO: Add check that valid lifetime must be >= 0
```

Comment:
```suggestion
			if lifetime < 0 {
				return d.Errf("lifetime must be > 0: %s", lifetime)
			}
```

Reply:
Just applied the suggested change with a small twist : the lifetime can be 0